.github/workflows/update-insecure-dependencies.yaml

```diff
@@ -7,27 +7,36 @@ on:
 env:
   IGNORE_PACKAGES: "github.com/aws/aws-sdk-go"
 jobs:
-  update-insecure-dependencies:
-    strategy:
-      fail-fast: false
-      matrix:
-        branch:
-          - "release-2.0"
-          - "release-1.8"
-          - "release-1.7"
-          - "release-1.6"
-          - "release-1.5"
-          - "master"
+  build-matrix:
     runs-on: ubuntu-latest
+    outputs:
+      branches: ${{ steps.generate-matrix.outputs.branches }}
+      token: ${{ steps.github-app-token.outputs.token }}
     steps:
       - name: Generate GitHub app token
         id: github-app-token
         uses: tibdex/github-app-token@021a2405c7f990db57f5eae5397423dcc554159c # v1.7.0
         with:
           app_id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
+      - name: generate-matrix
+        run: |
+          ACTIVE_BRANCHES=`gh api /repos/${{ github.repository }}/contents/active-branches.json --jq '.content | @base64d'`
```

Comment:
is it automatically unbase64-d?

Reply:
the jq command at the end has @base64d to decode